.doks/Makefile

```diff
@@ -14,4 +14,17 @@ deploy_to_doks:
 		--set global.cacheSecret=$${SHARED_PLAY_CRYPTO_SECRET} \
 		--set codacy-api.config.license=$${CODACY_LICENSE} \
 		--set global.codacy.url="http://k8s.dev.codacy.org" \
-		--set global.codacy.backendUrl="http://k8s.dev.codacy.org"
+		--set global.codacy.backendUrl="http://k8s.dev.codacy.org" \
+		--set listener.nfsserverprovisioner.storageClass.name="${RELEASE_NAME}-listener-cache-class" \
+		--set fluentdoperator.namespaces={"${RELEASE_NAME}"}
```

Comment:
same.. we should fix this in the fluend template so that it works like this by default

Reply:
fixed